packages/expo-camera/ios/EXCamera/EXCameraManager.m

```diff
@@ -73,6 +75,10 @@ - (NSDictionary *)constantsToExport
                @"cinematic": @(EXCameraVideoStabilizationModeCinematic),
                @"auto": @(EXCameraAVCaptureVideoStabilizationModeAuto)
                },
+           @"Permission": @{
+               @"all": @(EXCameraPermissionAll),
+               @"videoOnly": @(EXCameraPermissionVideoOnly),
+               },
```

Comment:
We don't have to export those values, do we? Especially that, we don't use them anyway. Or maybe I missed something?  
cc @bbarthec